repose-aggregator/core/repose-core/src/main/java/org/openrepose/powerfilter/PowerFilter.java

```diff
@@ -531,6 +495,12 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
         MDC.clear();
     }
 
+    private static void closeScope(Scope scope, Number value) {
+        scope.span().setTag(Tags.HTTP_STATUS.getKey(), value);
```

Comment:
To be consistent, maybe use the `Tag` to set the tag?
```
Tags.HTTP_STATUS.set(scope.span(), value);
```

Reply:
Maybe consistent isn't the right word, because it begs the question consistent with what... if anything, the way it is currently is consistent with the builder's call to `withTag(...)`. So I'll just throw that out as an option, but I won't go so far as to recommend it. Though, overall, we probably should be consistent.

Reply:
I'm all for being consistent, even if it is just within the project.
Let's make this an action item for standup.